app/bundles/PluginBundle/Form/Type/FieldsTypeTrait.php

```diff
@@ -166,6 +166,7 @@ function (FormEvent $event) use ($options, $integrationFields, $mauticFields, $f
                                 'attr'        => [
                                     'data-toggle' => 'tooltip',
                                     'title'       => 'mautic.plugin.direction.data.update',
+                                    'disabled'    => (isset($fieldData[$fieldsName][$field]) && $options['integration_object']->isCompoundMauticField($fieldData[$fieldsName][$field])) ? true : false,
```

Comment:
This could be simplified to

`isset($fieldData[$fieldsName][$field]) ? $options['integration_object']->isCompoundMauticField($fieldData[$fieldsName][$field]) : false`

But it's just a syntax sugar.